src/test/kotlin/org/jetbrains/research/transformations/SetupParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.transformations
+
+import java.io.File
+import java.util.logging.Logger
+
+object SetupParser {
+    val LOG = Logger.getLogger(this.javaClass.name)
+    private const val TARGET_PATH = "/tmp/pythonparser"
+
+    fun putParserToTmp() {
+        //copy parser file into /tmp directory, required by Gumtree
+        try {
+            LOG.info("Putting parser into $TARGET_PATH")
+
+            val pythonparserFile = File(javaClass.getResource("./parser/pythonparser.py").path)
+            val targetFile = File(TARGET_PATH)
+
+            pythonparserFile.copyTo(targetFile)
+            makeFileExecutable()
+        } catch (e: FileAlreadyExistsException) {
+            LOG.info("Parser file is already in $TARGET_PATH")
+        }
+
+    }
+
+    fun makeFileExecutable() {
+        LOG.info("Making parser file executable")
+        val targetFile = File(TARGET_PATH)
+        val command = arrayOf("chmod", "+x", targetFile.absolutePath)
+        val builder = ProcessBuilder(*command)
+        builder.directory(targetFile.parentFile)
+        builder.start()
+
+        // add pythonparser's path into system path
+        System.setProperty("gt.pp.path", TARGET_PATH)
```

Comment:
Not sure that we need to set property here because it's not about making file executable. Maybe we should move it to "putParserToTmp" because you handle all events about putting to tmp there and set TARGET_PATH as a system property is related to tmp?

Reply:
I agree with this and previous comments, thank you!